### PR TITLE
Updating TenantID information in the article based on customer input

### PR DESCRIPTION
**Pull request by LikhithaMS**
Though the article is clear with all of us being familiar with what a tenantID is, one of the customer ended up feeding the domain onmicrosoft.com name in the place on the $SourcetenantID and $TargettenantID looking at the following check it does against the domainname i.e. $existingOrgRel = $orgrels | ?{$_.DomainNames -like $targetTenantId}
So they have a request of adding a screenshot of an example  which depicts  $SourcetenantID and $TargettenantID.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/enterprise/cross-tenant-mailbox-migration.md [hunk 1]
```diff
@@ -200,7 +200,12 @@ To obtain the tenant ID of a subscription, sign in to the [Microsoft 365 admin c
        New-OrganizationRelationship "[name of your organization relationship]" -Enabled:$true -MailboxMoveEnabled:$true -MailboxMoveCapability RemoteOutbound -DomainNames $targetTenantId -OAuthApplicationId $appId -MailboxMovePublishedScopes $scope
    }
    ```
-
+   
+Note: TenantID fed to $sourceTenantId and $targetTenantId is the GUID and not the tenant domain name.
+To find your TenantID - https://docs.microsoft.com/en-us/onedrive/find-your-office-365-tenant-id
+Here is an example of Tenant ID
+![image](https://user-images.githubusercontent.com/74538523/145413901-e8924abf-13bd-442c-912d-932707ae8b9c.png)
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/cross-tenant-mailbox-migration.md]:
```suggestion
> [!NOTE]
> The tenant ID that you enter as the $sourceTenantId and $targetTenantId is the GUID and not the tenant domain name. For an example of a tenant ID and information about finding your tenant ID, see [Find your Microsoft 365 tenant ID](/onedrive/find-your-office-365-tenant-id).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @LikhithaMS, kindly commit the suggested changes if you agree with them (note that the screenshot is already included in the referenced article). Many thanks.

**Comment by LikhithaMS** [comment]
Done, thank you!

Regards,
Likhitha

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kelleyvice-msft Please merge this PR. Thanks

**Pull request merged**